content/maxwell3_fdem/emi_tutorial/amplitudeandphase.rst

```diff
@@ -0,0 +1,129 @@
+.. _amplitudeandphase:
+
+Amplitude and phase
+===================
+
+.. figure:: ./images/Concepts_3loops_only_K.png
+   :align: center
+   :scale: 60%
+   :name: Concepts_3loops_only_K
+
+   Conceptual diagram for 3-loops system.
+
+Based upon the induced current in the body (Loop2), in :ref:`induced_currents_body`, here we consider amplitude and phase of the secondary magnetic field (:math:`H^s`) normalized by primary magnetic field (:math:`H^p`), :math:`\frac{H^s}{H^p}`, measured at Rx (Loop3). For this, we first consider the primary magnetic field at body (Loop2), :math:`H^p_2`, resulting from a current, :math:`i^p`, flowing in the Loop1, which can be expressed as
```

Comment:
- "Based upon the induced current in the body (Loop 2), **as discussed in**..."
- I would also suggest mentioning the word "data" (ie. "data measured at the Rx (Loop3)") to be explicit